website/src/pages/markdown-page.md

```diff
@@ -1,7 +0,0 @@
----
-title: Markdown page example
----
-
-# Markdown page example
-
-You don't need React to write simple standalone pages.
```

Comment:
@bhasudha guess this was there from the beginning :) https://hudi.apache.org/markdown-page/